### ISSUE TITLE
Stuck at Party screen

### ISSUE THREAD

**Issue by erintesden** [Fate-Grand-Automata/FGA#988]
### Preparation

- [X] I read the [Troubleshooting Guide](https://github.com/MathewSachin/Fate-Grand-Automata/wiki/Troubleshooting)
- [X] My question isn't answered in any other issues (even the closed ones)

### Your question

Sometimes, at the party select screen it gets stuck.
Like, FGA tries to press Start on the party I have selected on the profile Im using and nothing happens. The same happens if I try to press start by myself, It forces me to go back a screen, or select another party and then go back to the one I actually want for the selection to work again.
This has happened to me both on my phone and while emulating the game.
Dont know if there's some kind of config I could change to avoid this

**Comment by reconman** [comment]
Does FGA ever get stuck if you let it run a quest multiple times on it's own?

If yes, do you have the play button somewhere other than in the bottom left?

If not, I'll need a video. For the other cases, FGA cannot deal with all screens as starting points, for example from the screen where the party is shown.

**Comment by erintesden** [comment]
> Does FGA ever get stuck if you let it run a quest multiple times on it's own?
> If yes, do you have the play button somewhere other than in the bottom left?
Yeah, It tends to happen on that screen when Im repeating a same quest.
And nop. I always leave the play button on the bottom left. So thats not the problem.
> If not, I'll need a video. For the other cases, FGA cannot deal with all screens as starting points, for example from the screen where the party is shown.
Im not sure of being able to make a video because it happens kinda randomly. Sometimes happens, sometimes not.
But about what you said, which screens are more recommended as starting points?

**Comment by reconman** [comment]
Recommended starting points are:
- When the quest to start is in the top right
- In the friend support selection screen
- At the start of the battle

**Comment by Toonv5** [comment]
FGO has a sneaky anti bot measure where it freezes the Start Quest button if you try to pick a party that's already selected. The best way around this is to not set a party in FGA, manually select the party you want at the start instead and let FGA loop that way.
It'll automatically reuse the last used party, so you just have to remember to change your party if you're changing nodes.

**Comment by d7c0** [comment]
Has the party selection feature (where you designate the party within battle configs) worked for anyone, or is it useless? I have the same problem. As Toonv5 said, it seems to block the "Start Quest" button

**Comment by erintesden** [comment]
> Has the party selection feature (where you designate the party within battle configs) worked for anyone, or is it useless? I have the same problem. As Toonv5 said, it seems to block the "Start Quest" button

Assuming the problem still persists in newer versions of the app (I havent updated it in almost a year). Then the feature is buggy and as someone recommended me, you should just circumvent it:
Configurate FGA to simply select the default party instead of an specific one.
And every time you are going to repeat an specific stage, choose the right party for it, and then initiate the FGA auto-play once the battle starts.

**Comment by reconman** [comment]
The bug only happens if you haven't cleared Solomon, it seems.

Haven't looked into it yet.

**Comment by d7c0** [comment]
> The bug only happens if you haven't cleared Solomon, it seems.
> 
> Haven't looked into it yet.

Had to finish P2 prologue for it to work
